### DISCUSSION TITLE
Can the evaluation continue after modifying the model configuration?  模型配置更改后，评测是否能够继续？

### DISCUSSION THREAD

**Discussion by niexufei** [open-compass/opencompass#378] [Q&A] [answered]
I want to evaluate a model, and first configure it using the API. The model's configuration file is as follows:

```
from opencompass.models.openai_api import OpenAI
models = [
    dict(abbr='wizardlm-13b-v12-api',
         path='WizardLM-13B-V1.2',
         type=OpenAI,
         key='1234',
         query_per_second = 1,
         openai_api_base = "http://10.234.9.38:31530/v1/chat/completions",
         max_out_len=200,
         max_seq_len=2048,
         batch_size=1
        )
]
```

There are a lot of datasets to evaluate, and during the inference phase, some of the dataset results have been generated, but not all of them have been completed.
At this point, if I want to modify the configuration of the model to use the Hugging Face Transformer for loading the model weights provided by Opencompass for inference, the configuration will be modified to:
```
models = [
    dict(
        type=HuggingFaceCausalLM,
        abbr='wizardlm-13b-v12',
        path="./models/WizardLM-13B-V1.2/",
        tokenizer_path='./models/WizardLM-13B-V1.2/',
        tokenizer_kwargs=dict(padding_side='left',
                              truncation_side='left',
                              use_fast=False,
                              ),
        max_out_len=100,
        max_seq_len=2048,
        batch_size=8,
        model_kwargs=dict(device_map='auto'),
        batch_padding=False, # if false, inference with for-loop without batch padding
        run_cfg=dict(num_gpus=4, num_procs=1),
    )
]
```
When running run.py with the --reuse parameter, is it possible to continue the evaluation from the last checkpoint and continue the evaluation for datasets that haven't been fully inferred yet?

**Comment by Leymore** [comment, marked as answer]
It will continue if you keep the `abbr` attribute in model config the same as the origin one.

**Reply by niexufei** [reply]
Thank you for your response, and thank you for adjusting the format of the question ：）
